src/components/InputField.vue

```diff
@@ -45,6 +45,7 @@ watch(
 );
 
 async function keyDown(key: KeyboardEvent) {
+  //NOSONAR
```

Comment:
I think line 63 to 76 of this function are useless and can be removed.

Maybe then it is also possible to not use `NOSONAR`